rocAL/rocAL/source/pipeline/node.cpp

```diff
@@ -70,3 +78,25 @@ Node::update_parameters()
 {
     update_node();
 }
+
+// Create vx_tensor for the crop coordinates
+void Node::create_crop_tensor(vx_tensor &crop_tensor, void **crop_coordinates) {
+    vx_size num_of_dims = 2;
+    vx_size stride[num_of_dims];
+    std::vector<size_t> crop_tensor_dims = {_batch_size, 4};
+    if(_inputs[0]->info().layout() == RocalTensorlayout::NFCHW || _inputs[0]->info().layout() == RocalTensorlayout::NFHWC)
```

Comment:
What if the layout is NCHW or NHWC, what would crop_tensor_dims be in that case?

Reply:
That has been handled in Line 86